### PR TITLE
Include NotificationConfig on BEGIN message for bolt scheme

### PR DESCRIPTION
**Pull request by injectives**
`NotificationConfig` might be ignored on new sessions with `bolt` scheme. This update fixes it.

### CODE REVIEW

File: bundle/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.neo4j.driver</groupId>
     <artifactId>neo4j-java-driver-parent</artifactId>
-    <version>5.8-SNAPSHOT</version>
+    <version>5.7-SNAPSHOT</version>
```
Comment (AndyHeap-NeoTech) [inline bundle/pom.xml:9]: Is this still valid if we are releasing in 5.8?

File: driver/pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.neo4j.driver</groupId>
     <artifactId>neo4j-java-driver-parent</artifactId>
-    <version>5.8-SNAPSHOT</version>
+    <version>5.7-SNAPSHOT</version>
```
Comment (AndyHeap-NeoTech) [inline driver/pom.xml:9]: Is this still valid if we are releasing in 5.8?

File: examples/pom.xml [hunk 3]
```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.neo4j.driver</groupId>
     <artifactId>neo4j-java-driver-parent</artifactId>
-    <version>5.8-SNAPSHOT</version>
+    <version>5.7-SNAPSHOT</version>
```
Comment (AndyHeap-NeoTech) [inline examples/pom.xml:9]: Is this still valid if we are releasing in 5.8?

**Review by AndyHeap-NeoTech — APPROVED** [review]
Just one question, is the change to the 5.7 snapshot still valid if the fix is going to go out in 5.8?

File: bundle/pom.xml (hunk 1 above)
Comment (injectives) [reply bundle/pom.xml:9]: It is fine, we will have a chance to update it later.

**Comment by injectives** [comment]
> Just one question, is the change to the 5.7 snapshot still valid if the fix is going to go out in 5.8?

https://github.com/neo4j/neo4j-java-driver/pull/1404#discussion_r1159509368

**Pull request merged**